bin/local.js

```diff
@@ -68,7 +69,10 @@ function onFileChange(evt, filepath) {
 }
 
 function startFileWatch() {
-    watcher.watch(config.sourceLocation, {
+    if (fswatcher){
+        fswatcher.close();
```

Comment:
Yeah, I think this is safe for now. Chokidar is pretty good at keeping up to date with fs changes (including newer files).

Let's roll with this for now since it's more explicit!